A/index.js

```diff
@@ -0,0 +1,81 @@
+const { request, response } = require("express")
+const express = require("express")
+const app  = express()
+app.use(express.json())
+
+
+let notes = [
```

Comment:
is notes used?

Reply:
Yes.
The task of the phone book is on ./A Phonebook